lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-vmware-infra_manager-vm.rb

```diff
@@ -15,5 +15,10 @@ def add_disk(disk_name, disk_size_mb, options = {})
     def remove_disk(disk_name, options = {})
       sync_or_async_ems_operation(options[:sync], "remove_disk", [disk_name, options])
     end
+
+    def move_into_folder(folder, options = {})
```

Comment:
Is options needed here?

Reply:
Well, it takes options -- https://github.com/ManageIQ/manageiq-providers-vmware/blob/78e3358127cc3542f8be1e3de9f7963759135d6a/app/models/manageiq/providers/vmware/infra_manager.rb#L265

Reply:
It is calling [this method](https://github.com/ManageIQ/manageiq/blob/23a1cc4ef6be627e3e4deeeb7a908923fd020150/app/models/vm_or_template/operations/relocation.rb#L109).